app/reducers/legalNotices/index.ts

```diff
@@ -20,26 +17,48 @@ export const storePrivacyPolicyClickedOrClosed = () => ({
   type: ACTIONS.STORE_PRIVACY_POLICY_CLICKED_OR_CLOSED,
 });
 
-export const shouldShowNewPrivacyToastSelector = (
-  state: RootState,
-): boolean => {
+// New selector for privacy policy feature flag
+export const selectPrivacyPolicyUpdateFeatureFlag = (state: RootState): string => {
+  const remoteFeatureFlags = selectRemoteFeatureFlags(state);
+  const policyValue = String(remoteFeatureFlags?.transactionsPrivacyPolicyUpdate || '');
+
+  if (policyValue === 'active_update') {
+    return '2025-04-01T12:00:00Z'; // A date in the past
+  }
+
+  return policyValue;
+};
+
+export const shouldShowNewPrivacyToastSelector = (state: RootState): boolean => {
   const {
     newPrivacyPolicyToastShownDate,
     newPrivacyPolicyToastClickedOrClosed,
   } = state.legalNotices;
 
+  // Get the feature flag value
+  const policyUpdateDate = selectPrivacyPolicyUpdateFeatureFlag(state);
+
+  // If feature flag is not set or empty, don't show toast
+  if (!policyUpdateDate) return false;
+
+  // If toast was already clicked or closed, don't show it again
   if (newPrivacyPolicyToastClickedOrClosed) return false;
```

Comment:
S: you could make both conditions in one, not impacting readability that much and making it clear that the outcome is similar:
```suggestion
  // If feature flag is not set or empty, don't show toast
  // If toast was already clicked or closed, don't show it again
  if (!policyUpdateDate || newPrivacyPolicyToastClickedOrClosed) return false;
```